### PR TITLE
Add live test targets

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
@pondrejk this is interesting for you I think

File: Makefile [hunk 1]
```diff
@@ -19,17 +19,19 @@ PYTEST = pytest -n 4 --boxed -v
...
+	@echo "  help             to show this message"
+	@echo "  info             to show infos about the collection"
+	@echo "  lint             to run code linting"
+	@echo "  test             to run unit tests"
+	@echo "  livetest         to run test playbooks live (without vcr)"
+	@echo "  sanity           to run santy tests"
+	@echo "  setup            to set up test, lint"
+	@echo "  test-setup       to install test dependencies"
+	@echo "  test_<test>      to run a specific unittest"
+	@echo "  livetest_<test>  to run a specific unittest"
```
Comment (pondrejk) [inline Makefile:31]: maybe we should say here what is the difference between the two

File: Makefile [hunk 2]
```diff
@@ -57,12 +59,18 @@ test-check-mode: $(MANIFEST) | tests/test_playbooks/vars/server.yml
...
 	$(PYTEST) -k 'not test_crud.py'
 
+livetest: $(MANIFEST) | tests/test_playbooks/vars/server.yml
+	pytest -v 'tests/test_crud.py::test_crud' --vcrmode live
+
 test_%: FORCE $(MANIFEST) | tests/test_playbooks/vars/server.yml
 	pytest -v 'tests/test_crud.py::test_crud[$*]' 'tests/test_crud.py::test_check_mode[$*]' $(FLAGS)
 
+livetest_%: FORCE $(MANIFEST) | tests/test_playbooks/vars/server.yml
+	pytest -v 'tests/test_crud.py::test_crud[$*]' --vcrmode live
```
Comment (pondrejk) [inline Makefile:69]: could we have $(FLAGS) here as well?

**Comment by evgeni** [comment]
@mdellweg do you want to incorporate the comments from Peter in a new PR?

File: Makefile (hunk 2 above)
Comment (mdellweg) [reply Makefile:69]: I guess so... PR will follow.

**Pull request merged**